build.sbt

```diff
@@ -40,13 +40,13 @@ lazy val nativeLibraryPath = {
 }
 
 val distBuildDir = file(".") / "target" / "dist" / "polynote"
-val scalaVersions = Seq("2.11.12", "2.12.12", "2.13.6")
+val scalaVersions = Seq("2.11.12", "2.12.15", "2.13.6")
 lazy val scalaBinaryVersions = scalaVersions.map {
   ver => ver.split('.').take(2).mkString(".")
 }.distinct
 
 val commonSettings = Seq(
-  scalaVersion := "2.11.12",
+  scalaVersion := "2.12.15",
```

Comment:
This is changing the default version of Polynote to 2.12, right? Are there any docs, configs, etc. that need to be updated to reflect that? Do you think we can stop publishing for 2.11? 🙏

Reply:
Oh, I only did that so that IntelliJ would see the 2.12 code and dependencies. That's pretty much the only thing this controls.